### PR TITLE
feat: make errors prettier

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

We already gather `$exception` events using the Sentry integration and now can gather the same events directly from posthog-js

But we don't treat them specially in the UI

## Changes

* Make them a little prettier in the data table 
* Make them a little prettier in the recording player list
* Add a mini filter for exception events in the player list controls

## How did you test this code?

👀 locally

### CODE REVIEW

File: frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss [hunk 1]
```diff
@@ -108,6 +108,11 @@
             &:not(:first-child) {
                 border-top: 1px solid var(--border);
             }
+            // some "row types" need to override defaults
+            // here, the $exception event needs to override border color
```
Comment (pauldambra) [inline frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss:112]:
```suggestion
```

File: frontend/src/lib/lemon-ui/LemonTable/types.ts [hunk 2]
```diff
@@ -45,6 +45,10 @@ export type LemonTableColumns<T extends Record<string, any>> =
 export interface ExpandableConfig<T extends Record<string, any>> {
     /** Row expansion render function. */
     expandedRowRender: (record: T, recordIndex: number) => any
+    /* className string or function that generates one
```
Comment (pauldambra) [inline frontend/src/lib/lemon-ui/LemonTable/types.ts:48]:
```suggestion
    /* className string, or function that generates one,
```

File: frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts [hunk 3]
```diff
@@ -67,6 +67,12 @@ const MiniFilters: SharedListMiniFilter[] = [
         name: 'Autocapture',
         tooltip: 'Autocapture events such as clicks and inputs',
     },
+    {
+        tab: SessionRecordingPlayerTab.EVENTS,
+        key: 'events-exceptions',
+        name: 'Exceptions',
+        tooltip: 'Exception events from PostHog or its Sentry integration',
+    },
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts:75]: I like this _but_ I worry that adding it without having our own exception tracking will confuse people... 
I guess we can see if anyone clicks it and then maybe think about adding a helpful empty state like "You're project has no exception events - here is what you can do to enable them"

**Review by benjackwhite — APPROVED** [review]

File: frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts (hunk 3 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts:75]: Ah, good call!

**Pull request merged**